.github/workflows/python_package.yml

```diff
@@ -118,6 +121,35 @@ jobs:
             compiler: gcc
             method: wheel
             python_version: '3.10'
+          - os: ubuntu-latest
+            container: lightgbm.azurecr.io/vsts-agent:manylinux_2_28_x86_64
+            task: mpi
+            compiler: gcc
+            method: source
+            python_version: '3.10'
+            in-ubuntu-base-container: 'false'
+            setup-conda: 'false'
+          - os: ubuntu-latest
+            container: &ubuntu_latest_image mcr.microsoft.com/mirror/docker/library/ubuntu:22.04
```

Comment:
```suggestion
            container: &ubuntu_latest_image mcr.microsoft.com/mirror/docker/library/ubuntu:22.04
            os-display-name: 'ubuntu22.04'
```

Reply:
Let's use YAML anchors for this too, so that when we move "latest Linux" to Ubuntu 24.04 or something in the future, we don't also have to update all these lines. I'll push a commit that does that.